### ISSUE TITLE
[N7 Account Settings] Create Contact Method Details page

### ISSUE THREAD

**Issue by Beamanator** [Expensify/App#11575]
Tracking issue: https://github.com/Expensify/Expensify/issues/182122

Detailed section of doc: https://docs.google.com/document/d/17akqrbC-w8FqEoDLvcLmrX6iw4uaVrT_xlwAl-IhEzE/edit#heading=h.vwu67pkf1q06

Note: This can only be completed after all API commands in this issue have been created: https://github.com/Expensify/Expensify/issues/231921
<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~014ac2d7133fe25e10</li>
        <li>Upwork Job ID: 1625425719555817472</li>
        <li>Last Price Increase: 2023-02-14</li>
    </ul>
</details>

**Comment by Beamanator** [comment]
On hold for WhatsApp Quality

**Comment by Beamanator** [comment]
Still on hold for now, might pick this up next week as discussions from here (https://github.com/Expensify/App/issues/11795) are indicating this could be useful to implement sooner rather than later

**Comment by Beamanator** [comment]
Unassigning since k2 is not liking that i have so many issues assigned to me on hold, will get back to this once N7 Account Settings if off hold (waiting on #passwordless too probably)

**Comment by Beamanator** [comment]
Monthly is fine, still waiting for #passwordless to wrap up

**Comment by MelvinBot** [comment]
@Beamanator, this ***Monthly*** task hasn't been acted upon in 6 weeks; closing.

If you disagree, feel encouraged to reopen it -- but pick your least important issue to close instead.

**Comment by Beamanator** [comment]
No longer on hold for passwordless!

**Comment by MelvinBot** [comment]
Triggered auto assignment to Contributor Plus for review of internal employee PR - @mollfpr (`Internal`)

**Comment by Beamanator** [comment]
Oops we're not working on this one quite yet, but will soon 👍

**Comment by Beamanator** [comment]
PR is live and ready for review!

**Comment by mollfpr** [comment]
Thanks, @Beamanator! I will review it after the clarity in the other issue.

**Comment by Beamanator** [comment]
@mollfpr actually if you don't mind reviewing no matter what that would be great - since you're the one who got auto assigned on the PR 😅

**Comment by MelvinBot** [comment]
`Reviewing` label has been removed, please complete the "BugZero Checklist".

**Comment by MelvinBot** [comment]
The solution for this issue has been :rocket: deployed to production :rocket: in version 1.2.80-2 and is now subject to a 7-day regression period :calendar:. Here is the list of pull requests that resolve this issue:

- https://github.com/Expensify/App/pull/15204


If no regressions arise, payment will be issued on 2023-03-16. :confetti_ball:

After the hold period is over and BZ checklist items are completed, please complete any of the applicable payments for this issue, and check them off once done.
- [ ] External issue reporter
- [ ] Contributor that fixed the issue
- [ ] Contributor+ that helped on the issue and/or PR

As a reminder, here are the bonuses/penalties that should be applied for any External issue:
- Merged PR within 3 business days of assignment - 50% **bonus**
- Merged PR more than 9 business days after assignment - 50% **penalty**

**Comment by Beamanator** [comment]
FYI the PR for this issue got reverted in this PR (https://github.com/Expensify/App/pull/15761) due to this bug that it caused: https://github.com/Expensify/App/issues/15725

Working on the fix here: https://github.com/Expensify/App/pull/15751

**Comment by Beamanator** [comment]
Gunna close this in favor of https://github.com/Expensify/App/issues/15203

**Comment by MelvinBot** [comment]
⚠️ Looks like this issue was linked to a **Deploy Blocker** [here](https://github.com/Expensify/App/issues/15725)

If you are the assigned CME please investigate whether the linked PR caused a regression and leave a comment with the results.

If a regression has occurred and you are the assigned CM follow the instructions [here](https://stackoverflow.com/c/expensify/questions/13647).

If this regression could have been avoided please consider also proposing a recommendation to the PR checklist so that we can avoid it in the future.

**Comment by MelvinBot** [comment]
`Reviewing` label has been removed, please complete the "BugZero Checklist".

**Comment by MelvinBot** [comment]
The solution for this issue has been :rocket: deployed to production :rocket: in version 1.2.92-2 and is now subject to a 7-day regression period :calendar:. Here is the list of pull requests that resolve this issue:

- https://github.com/Expensify/App/pull/15751


If no regressions arise, payment will be issued on 2023-04-07. :confetti_ball:

After the hold period is over and BZ checklist items are completed, please complete any of the applicable payments for this issue, and check them off once done.
- [ ] External issue reporter
- [ ] Contributor that fixed the issue
- [ ] Contributor+ that helped on the issue and/or PR

As a reminder, here are the bonuses/penalties that should be applied for any External issue:
- Merged PR within 3 business days of assignment - 50% **bonus**
- Merged PR more than 9 business days after assignment - 50% **penalty**
